shanoir-ng-datasets/src/main/java/org/shanoir/ng/studycard/dto/StudyCardOnStudyResult.java

```diff
@@ -0,0 +1,80 @@
+package org.shanoir.ng.studycard.dto;
+
+/**
+ * This class contains the result of an application of a study card
+ * on an entire study. For each examination, when the result is wrong
+ * on the examination level already, we will not check deeper on the
+ * acquisition level. When everything is clear on the examination level,
+ * we check deeper on the acquisition level, if there is an error, we stop.
+ * Same for the dataset level.
+ * 
+ * In the idea, that it does not make sense to display an error on a dataset,
+ * when there is already a sequence missing on the exam. Display, that at first
+ * and then display the dataset result, when that problem solved?
+ * 
+ * @author mkain
+ *
+ */
+public class StudyCardOnStudyResult {
+
+	private String subjectName;
```

Comment:
Remark : If we want to provide links to datasets, acquisitions or examination in the GUI, we would have to add ids here.

Reply:
I agree, I will do so